libs/designsystem/src/lib/scss/_global-styles.scss

```diff
@@ -96,6 +96,25 @@ ion-modal.kirby-overlay {
       --border-radius: #{$border-radius};
       --box-shadow: #{get-elevation(8)};
 
+      &.kirby-drawer {
+        align-items: flex-end;
+      }
```

Comment:
Is this needed? We check explicitly for `:not(.kirby-drawer)` when we apply `align-items: flex-start;`...

Reply:
This is to ensure the drawer is placed at the bottom of the page instead of centered as before (sorry forgot to mention that)